rudder-web/src/main/scala/com/normation/plugins/RudderPlugin.scala

```diff
@@ -142,6 +142,64 @@ trait RudderPluginDef {
   def version : PluginVersion
 
 
+  /*
+   * Information about the plugin activation status
+   * and license information.
+   * In implementation, by default use
+   * com.normation.plugins.AlwaysEnabledPluginStatus
+   */
+  def status: PluginStatus
+
+  /*
+   * A visual representation of status/license information, with a
+   * default (overridable) rendering
+   */
+  def statusInformation(): NodeSeq = {
+    import net.liftweb.util.Helpers._
+
+    val info = (
+      <div class="tw-bs"><div id="license-information" style="padding:5px; margin: 5px;" class="bs-callout">
+        <h4>License information</h4>
+        <p>This binary version of the plugin is submited to a license with the following information:</p>
```

Comment:
This binary version of this plugin is suject to license with the following information